src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -141,14 +141,23 @@ public static float CalculateCapacity(IEnumerable<OrganelleTemplate> organelles)
 
     // TODO: maybe this should return a ValueTask as this is getting pretty computation intensive
     public static float CalculateSpeed(IReadOnlyList<OrganelleTemplate> organelles, MembraneType membraneType,
-        float membraneRigidity, bool isBacteria)
+        float membraneRigidity, bool isBacteria, bool useEstimate = false)
     {
+        float shapeMass = 0;
+
         // This is pretty expensive as we need to generate the membrane shape and *then* the collision shape to figure
-        // out the mass. We rely on the caches working extra hard here to ensure reasonable performance.
```

Comment:
Second part of this comment is cut out, I think it should be put back on the 
`            var membraneShape = MembraneComputationHelpers.GetOrComputeMembraneShape(organelles, membraneType);` line.